ios/Podfile.lock

```diff
@@ -638,4 +638,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ca4eb736278d4211ffaa5cef62299ef8fc2137ca
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```

Comment:
I think this should be reverted

Reply:
alright but it should be updated

Reply:
Should be updated together with gems :) So yes, please do it in a separate PR